cgi-bin/DW/Controller/Media.pm

```diff
@@ -68,14 +77,39 @@ sub media_bulkedit_handler {
 
         if ( $post_args->{"action:edit"} ) {
             my %post = %{$post_args->as_hashref||{}};
-            while ( my ($key, $secval) = each %post ) {
-                next unless $key =~ m/^security-(\d+)/;
-                my $mediaid = $1 >> 8;
-                my $media = DW::Media->new( user => $rv->{u}, mediaid => $mediaid );
+
+            # transform our HTML field names to property names
+            # and group by what media object they belong to
+            # we don't care if the id or prop might not exist
+            # right now, later steps will verify them
+
+            my %props;
+            while ( my ($key, $val) = each %post ) {
+                next if $key =~ m/delete/;
```

Comment:
Yeah, I checked and you can change this to `next if $key eq 'delete';`

Never use regular expressions where simple equality checks will do, especially if they're not anchored :)